projects/libcoap/project.yaml

```diff
@@ -0,0 +1,2 @@
+homepage: "https://libcoap.net/"
+primary_contact: "bergmann@tzi.org"
```

Comment:
Is this associated with google account, if not add google account email as well in auto cc field, so that you can access clusterfuzz reports.

Reply:
@bshastry Could you please add libcoap@gmail.com in the auto cc field? I just have created it for this purpose.